legacy/selectors.d.ts

```diff
@@ -27,10 +27,12 @@ export var easyConfirmationSelector: (state: State) => boolean
 export var isHWSelector: (state: State) => boolean
 export var walletNameSelector: (state: State) => string
 export var walletNamesSelector: (state: State) => Array<string>
+export var defaultNetworkAssetSelector: (state: State) => Token
```

Comment:
this returns `DefaultAsset`

Reply:
Yap, it returns from the `availableAssetsSelector` selector which return type is not precise, since it can return Token | DefaultAsset.